docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -930,6 +930,11 @@ for the data files and partition the storage per day using the column
 
     WITH ( format = 'ORC', partitioning = ARRAY['event_date'] )
 
+By default, the storage table is created in the same schema as the materialized
+view definition. The ``iceberg.materialized-views.storage-schema`` catalog
```

Comment:
Add this to the `General configuration` table as well, please

Reply:
done